pkgs/development/compilers/julia/generic.nix

```diff
@@ -48,29 +60,60 @@ stdenv.mkDerivation rec {
     patchShebangs .
   '';
 
+
   makeFlags = [
     "prefix=$(out)"
+  ] ++ lib.optionals stdenv.isDarwin (
+    let
+      macosProductVersion = pkgs.runCommand "macos-product-version" { } ''
+        echo "Running sw_vers..."
+        echo -n $(${darwin.DarwinTools}/bin/sw_vers -productVersion) > $out
+        echo "macOS product version: $(cat $out)"
+      '';
+      macosProductVersionStr = builtins.readFile "${macosProductVersion}";
+      macosPlatformVersion = pkgs.runCommand "macos-platform-version" { } ''
+        echo "Running xcrun..."
+        echo -n $(${pkgs.xcbuild}/bin/xcrun --show-sdk-version) > $out
+        echo "macOS platform version: $(cat $out)"
+      '';
+      macosPlatformVersionStr = builtins.readFile "${macosPlatformVersion}";
```

Comment:
This is IFD (import from derivation), which is generally not allowed in nixpkgs.

Reply:
Thanks, I see. Any tips on getting this info through? Otherwise we could alway go back to hardcoding it I suppose.